src/olympia/lib/crypto/tasks.py

```diff
@@ -8,68 +8,37 @@
 from olympia.addons.models import AddonUser
 from olympia.amo.celery import task
 from olympia.files.utils import update_version_number
-from olympia.lib.crypto.signing import SIGN_FOR_APPS, sign_file
+from olympia.lib.crypto.signing import sign_file
+from olympia.addons.models import Addon
 from olympia.versions.compare import version_int
 from olympia.versions.models import Version
 
 
 log = olympia.core.logger.getLogger('z.task')
 
 
-MAIL_SUBJECT = u'Mozilla Add-ons: {addon} has been automatically signed on AMO'
-MAIL_MESSAGE = u"""
-Your add-on, {addon}, has been automatically signed for distribution in
-upcoming versions of Firefox. The signing process involved re-packaging add-on
-files and changing the version string to ensure automatic updates work
-correctly. The new versions have kept their review status and are now available
-for your users.
+MAIL_COSE_SUBJECT = (
+    u'Your Firefox extension has been re-signed with a stronger signature')
 
-We recommend that you give them a try to make sure they don't have any
-unexpected problems: {addon_url}
+MAIL_COSE_MESSAGE = u'''
+Hello,
 
-If you are unfamiliar with the extension signing requirement, please read the
-following documents:
+Mozilla has recently upgraded the signing for Firefox extensions and themes to
```

Comment:
`extensions` is probably wrong here, that will affect themes and langpacks as well, right ? This is why the original message said `add-on`.

Reply:
It's saying "extensions and themes" but you're right, language packs are affected as well. cc @caitmuenster on this

Reply:
Wow I'm blind. The problem is present on the line below though, it says `all extension versions`.

Reply:
dictionaries are affected as well imho

Reply:
Question is, does "Extension" in a wider sense include themes and the others too? Not sure about our general wording in this regard.

Reply:
Historically it didn't, at least...

Reply:
Good points & thanks for the catch! Made a few tweaks here: https://docs.google.com/document/d/167MXie1DEXDVEM0j2V8ALC8MBSjjKqNp6Ebd9TV9gMg/edit#